File: testdata/RenameField/Java/parent/Employee.java
import java.util.Date;

public class Employee {
    private String name;
    private int age;
    private Date joinDate;

    public Employee(String name, int age, Date joinDate) {
        this.name = name;
        this.age = age;
        this.joinDate = joinDate;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    public Date getJoinDate() {
        return joinDate;
    }
}

File: testdata/RenameField/Cpp/parent/Employee.cpp
#include <string>
#include <ctime>

class Employee {
private:
    std::string name;
    int age;
    std::tm joinDate;

public:
    Employee(std::string name, int age, std::tm joinDate)
        : name(name), age(age), joinDate(joinDate) {}

    std::string getName() {
        return name;
    }

    int getAge() {
        return age;
    }

    std::tm getJoinDate() {
        return joinDate;
    }
};
